src/Jms/YamlConverter.php

```diff
@@ -35,7 +35,7 @@ public function __construct(NamingStrategy $namingStrategy)
             return "GoetasWebservices\Xsd\XsdToPhp\XMLSchema\Time";
         });
         $this->addAliasMap("http://www.w3.org/2001/XMLSchema", "date", function (Type $type) {
-            return "DateTime<'Y-m-d'>";
+            return "GoetasWebservices\Xsd\XsdToPhp\XMLSchema\Date";
```

Comment:
because a xs:date in the format 2002-01-01Z or 2002-01-01+01:00 or 2002-01-01-01:00 is a valid date string ;)